am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetRolePermissionsIntegrationTest.java

```diff
@@ -42,13 +45,23 @@ void setUp() {
 
     @Test
     void whenGettingRolePermissionsShouldReturnPermissionsAndSecurityClassificationsForSpecifiedRole() {
+
+        ImmutableSet multiPermission = ImmutableSet.builder().add(READ).add(CREATE).add(UPDATE).build();
+
         Map.Entry<Set<Permission>, SecurityClassification> publicReadPermission =
             new Pair<>(ImmutableSet.of(READ), PUBLIC);
 
-        addRoleWithSecurityClassification(roleName, PUBLIC);
+        Map.Entry<Set<Permission>, SecurityClassification> privateReadPermission =
+            new Pair<>(ImmutableSet.of(READ), PRIVATE);
+
+        Map.Entry<Set<Permission>, SecurityClassification> restrictedReadPermission =
```

Comment:
if you keep your multiPermissions this is no longer a restrictedReadPermission

Reply:
yes right

Reply:
done